Makefile

```diff
@@ -572,4 +576,6 @@ package:  ## Build rpm packages
 	./contrib/build_rpm.sh
 
 package-install: package  ## Install rpm packages
-	sudo ${PKG_MANAGER} -y install --allowerasing ${HOME}/rpmbuild/RPMS/*/*.rpm
```

Comment:
`--allowerasing` does not exist for yum, but cannot use swap as it will fail if it is the first time we install `podman` package